core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -386,6 +386,8 @@ object SharedSQLContext extends Logging {
       tpchDBName = getOrElse(prop, TPCH_DB_NAME, "tpch_test")
       tpcdsDBName = getOrElse(prop, TPCDS_DB_NAME, "")
 
+      val enableTiFlashTest = getOrElse(prop, ENABLE_TIFLASH_TEST, "false")
```

Comment:
can we just make it a global property for all tests?

Reply:
`enableTiFlashTest` can be used as a flag in test only. It does not need to be contained in `spark.conf`

Reply:
done